### PR TITLE
Bug 1963684: Return zero for DiskSize for OCS GetHostRequirements

### PR DESCRIPTION
**Pull request by rewantsoni**
OCS has no requirements towards installation diskSize

Closes: [OCPBUGSM-29504](https://issues.redhat.com/browse/OCPBUGSM-29504)

Signed-off-by: Rewant Soni <resoni@redhat.com>

### CODE REVIEW

**Review by nbalacha — APPROVED** [review]

**Review by carbonin — APPROVED** [review]

**Comment by nbalacha** [comment]
@rewantsoni , this looks like a genuine test failure. Please check the following and see if it should be removed
https://github.com/openshift/assisted-service/blob/master/subsystem/cluster_test.go#L3213

File: subsystem/cluster_test.go [hunk 1]
```diff
@@ -3168,76 +3167,6 @@ var _ = Describe("cluster install, with default network params", func() {
 		})
 	})
 })
-var _ = Describe("Update cluster with OLM operators", func() {
```
Comment (nbalacha) [inline subsystem/cluster_test.go:3171]: Don't remove the entire function - only the part dealing with OCS

Comment (rewantsoni) [reply subsystem/cluster_test.go:3171]: @jakub-dzon suggested that we can remove the test as it's no longer valid

Comment (nbalacha) [reply subsystem/cluster_test.go:3171]: Ack.

**Review by nbalacha — APPROVED** [review]

**Comment by nbalacha** [comment]
@gamli75

**Pull request merged**